### PR TITLE
Fix padding in featured books more links and notes in content 

### PR DESCRIPTION
**Pull request by amberwebb**
When viewed in portrait mode on mobile

### CODE REVIEW

File: src/scripts/modules/media/body/body.less [hunk 1]
```diff
@@ -40,7 +40,12 @@
     @import 'content.less';
   }
 
-  cnx-pi {
+   cnx-pi {
```
Comment (dak) [inline src/scripts/modules/media/body/body.less]: I don't think this space was intentional?

File: src/scripts/modules/media/footer/footer.less [hunk 2]
```diff
@@ -4,6 +4,8 @@
 .media-footer {
   margin: 4rem 6rem 6rem 6rem;
 
+  @media
```
Comment (dak) [inline src/scripts/modules/media/footer/footer.less]: Is this intended to be here?

**Comment by dak** [comment]
The build failure on the push appears to be because it's building on travis' legacy hardware, as opposed to the pr build which is in a container.  Not sure why that may be the case (maybe rebase from master?), but either way it appears that it can be ignored.

File: src/scripts/modules/media/footer/footer.less (hunk 2 above)
Comment (amberwebb) [inline src/scripts/modules/media/footer/footer.less]: I pulled this code out from the stash from a month or so ago. This is where I left off, so no it was not intended to be there like that.

**Pull request merged**